src/chainstate/stacks/index/bits.rs

```diff
@@ -324,33 +257,36 @@ pub fn read_hash_bytes<F: Read + Seek>(f: &mut F, buf: &mut Vec<u8>) -> Result<(
             }
         })?;
     
-    fast_extend_from_slice(buf, &hashbytes);
-    Ok(())
+    Ok(hashbytes)
+}
+
+pub fn read_4_bytes<F: Read + Seek>(f: &mut F) -> Result<[u8; 4], Error> {
```

Comment:
Can we make this name more specific?  Is it only used for reading the block ID?  If so, can we call it `read_block_id` or something?

Reply:
Addressed in d31f0207